Remove all log files beyond $ROTATE

rhodesn opened the pull request:
For rackerlabs/auter/issues/110, if $ROTATE is reduced then auter should
clean up all remaining log files greater than $ROTATE.

Also declare OUTPUT_FILE as local and change to lowercase as per Google style
guide.

reaperzn reviewed (changes requested):
@nickrhodes-gh Please resolve the conflicts that exist.

Please also pull the latest changes from the develop branch which will resolve most of the travis errors.

Note the Travis error on line 82: This should be changed to $((i+1)) instead of $(($i+1))

Please also revert to using caps for the OUTPUT_FILE variable. As per https://google.github.io/styleguide/shell.xml#Variable_Names, there does not seem to be any definition saying that the variables should be lower case. There is a definition for function names which we do adhere to however variables in the auter script are all in caps. As per the Conclusion in that guide, you are directed to the "Parting Words" in https://google.github.io/styleguide/cppguide.html#Parting_Words which state:
"local style is also important" and that these are guidelines with the objective being consistency.

nickrhodes-gh commented:
I'll change the output_file variable to uppercase, however the style guide shows local variables defined in lowercase and explicitly states global variables should be uppercase.

rhodesn commented:
Closed/re-opened to trigger new build.

rhodesn committed:
Remove old output files greater than ROTATE

If ROTATE is reduced then we need to clean up the older output files
that are no longer within the threshold.

rhodesn commented:
Closing to reimplement this.
